bin/all_sky_search/pycbc_add_statmap

```diff
@@ -415,7 +422,7 @@ bg_coinc_type = np.array([])
 for f_in in files:
     key = get_ifo_string(f_in).replace(' ','')
     combo_repeat_fg = np.array(np.repeat(key.encode('utf8'),
-                               f_in['foreground/fap'].size))
+                               f_in['foreground/stat'].size))
```

Comment:
what is this change doing?

Reply:
This is to match what is done earlier in the code for the fg_coinc_type.

Simply for consistency, as it made me think "why FAP hereand stat earlier?"